sdk/cosmosdb/cosmos/test/public/functional/clientSideEncryption.spec.ts

```diff
@@ -0,0 +1,2346 @@
+      testKeyVault,
+      "key1",
+      "tempmetadata1",
+      KeyEncryptionKeyAlgorithm.RSA_OAEP,
+    );
+    metadata2 = new EncryptionKeyWrapMetadata(
+      testKeyVault,
+      "key2",
+      "tempmetadata2",
+      KeyEncryptionKeyAlgorithm.RSA_OAEP,
+    );
+    encryptionClient = new CosmosClient({
+      endpoint: endpoint,
+      key: masterKey,
+      enableEncryption: true,
+      keyEncryptionKeyResolver: testKeyEncryptionKeyResolver,
+      encryptionKeyResolverName: testKeyVault,
+      encryptionKeyTimeToLive: EncryptionTimeToLive.NoTtl(),
+    });
+    database = (await encryptionClient.databases.createIfNotExists({ id: randomUUID() })).database;
+    const revokedKekMetadata = new EncryptionKeyWrapMetadata(
+      testKeyVault,
+      "revokedKek",
+      "revokedKek-metadata",
+      KeyEncryptionKeyAlgorithm.RSA_OAEP,
+    );
+    await testCreateClientEncryptionKey(database, "key1", metadata1);
+    await testCreateClientEncryptionKey(database, "key2", metadata2);
+    await testCreateClientEncryptionKey(database, "keyWithRevokedKek", revokedKekMetadata);
+    const key1Paths = [
+      "/PK",
+      "/sensitive_NestedObjectFormatL1",
+      "/sensitive_IntFormat",
+      "/sensitive_DateFormat",
+      "/sensitive_FloatFormat",
+      "/sensitive_ArrayMultiTypes",
+    ].map(
+      (path) =>
+        new ClientEncryptionIncludedPath(
+          path,
+          "key1",
+          EncryptionType.DETERMINISTIC,
+          EncryptionAlgorithm.AEAD_AES_256_CBC_HMAC_SHA256,
+        ),
+    );
+    const key2Paths = [
+      "/id",
+      "/sensitive_DecimalFormat",
+      "/sensitive_BoolFormat",
+      "/sensitive_IntMultiDimArray",
+      "/sensitive_ObjectArrayType",
+    ].map(
+      (path) =>
+        new ClientEncryptionIncludedPath(
+          path,
+          "key2",
+          EncryptionType.DETERMINISTIC,
+          EncryptionAlgorithm.AEAD_AES_256_CBC_HMAC_SHA256,
+        ),
+    );
+    const paths = key1Paths.concat(key2Paths);
+    paths.push(
+      new ClientEncryptionIncludedPath(
+        "/sensitive_ArrayFormat",
+        "keyWithRevokedKek",
+        EncryptionType.DETERMINISTIC,
+        EncryptionAlgorithm.AEAD_AES_256_CBC_HMAC_SHA256,
+      ),
+    );
+    clientEncryptionPolicy = new ClientEncryptionPolicy(paths, 2);
+    containerDefinition = {
+      id: randomUUID(),
+      partitionKey: {
+        paths: ["/PK"],
+      },
+      clientEncryptionPolicy: clientEncryptionPolicy,
+    };
+
+    const containerDefinitionForChangeFeed = {
+      id: randomUUID(),
+      partitionKey: {
+        paths: ["/PK"],
+      },
+      clientEncryptionPolicy: clientEncryptionPolicy,
+    };
+    encryptionContainer = (await database.containers.createIfNotExists(containerDefinition))
+      .container;
+    encryptionContainerForChangeFeed = (
+      await database.containers.createIfNotExists(containerDefinitionForChangeFeed)
+    ).container;
+
+    await encryptionContainer.initializeEncryption();
```

Comment:
any specific reason for initializing separately? we have enable encryption flag in the cosmos client